src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
But react doesn't auto bind anymore, right?
I haven't tried yet but it's either that or bind it in the render (which means binding on each re-render, and that would be a waste of perfs 😢 )